src/docs/backup.dox

```diff
@@ -118,7 +118,8 @@ are no longer part of the backup when later incremental backups no longer
 return their name. This is especially important for WiredTiger log files
 that are no longer needed and must be removed before recovery is run.
 
-@copydoc doc_bulk_durability
+Block-based incremental backup can be performed after a bulk load
```

Comment:
Hmm I'm not sure if we should add brackets here, how about `Block-based incremental backup can be performed after a bulk load, without intervening with a full backup.`

Reply:
updated in [5040114](https://github.com/wiredtiger/wiredtiger/pull/7566/commits/50401141ef682def18ca997763da4e41dbe7e1d6)